op-node/params/constants.go

```diff
@@ -0,0 +1,7 @@
+package params
+
+const (
+	// Number of L1 blocks between when a channel can be opened and when it must be closed by.
+	ChannelTimeoutBedrock uint64 = 300
```

Comment:
It's not safe for any frontier chains that override this.  I don't think we should be concerned about them. This is worth a callout in the next op-stack release.
Ideally, we want the op-node to bail loudly if when it encounters a channel_timeout config. This will be done once we've removed the channel_timeout from the superchain-registry. After that, any frontier chain that relies on a custom channel timeout will immediately know when starting their op-node that their custom configs are no longer supported.

Reply:
We shouldn't break them in this case imo, we should just make the rollup config be optional and override the pre granite value if it exists

Reply:
Made changes to make the pre-granite channel timeout customizable again.